docs/user/reporting/index.asciidoc

```diff
@@ -24,47 +25,66 @@ runs on the server in headless mode to load {kib} and capture the rendered {kib}
 
 Chromium is an open-source project not related to Elastic, but the Chromium binary for {kib} has been custom-built by Elastic to ensure it
 works with minimal setup. However, the {kib} server OS might still require additional dependencies for Chromium. See the
-<<reporting-troubleshooting-system-dependencies, Reporting Troubleshooting>> section for more information about the system dependencies
+<<reporting-troubleshooting-system-dependencies, Reporting troubleshooting>> section for more information about the system dependencies
 for different operating systems.
 
 [float]
-== Manually generate reports
+[[reporting-required-privileges]]
+== Roles and privileges
 
-. Open {kib} in your web browser and log in. If you are running {kib}
-locally, go to `http://localhost:5601`. To access {kib} and generate
-reports, you need the `reporting_user` role, and an additional role with succifient <<kibana-privileges, Kibana privileges>>, such as the `kibana_user` role.
-For more information, see <<secure-reporting>>.
+To generate a report, you mut have the `reporting_user` role. You also need
+the appropriate <<kibana-privileges, {kib} privileges>> to access the objects that you
+want to report on and the {es} indices.
 
-. Open the dashboard, visualization, or saved search you want to include
-in the report.
+[float]
+[[manually-generate-reports]]
+== Generate a report manually
+
+. Open the dashboard, visualization, Canvas workpad, or saved search that you want to include in the report. 
+
+. In the {kib} toolbar, click *Share*.  If you are working in Canvas, 
+click the share icon image:user/reporting/images/canvas-share-button.png["Canvas Share button"].
 
-. Click *Share* in the {kib} toolbar:
+. Select the appropriate option for generating the report:
```

Comment:
Is this supposed to be a colon? Maybe a period would be better.

Reply:
Maybe reformat this to:

. Select the appropriate option for generating the report. You can export a dashboard or visualization as a:
* PNG or PDF document
* Canvas workpad as a PDF document
* Saved search as a CVS document